packages/react-api/src/hooks/useCartoLayerProps.js

```diff
@@ -1,8 +1,9 @@
 import { MAP_TYPES, API_VERSIONS } from '@deck.gl/carto';
 import { useSelector } from 'react-redux';
-import useGeoJsonFeatures from './useGeoJsonFeatures';
-import useTilesetFeatures from './useTilesetFeatures';
-import { getDataFilterExtensionProps } from './dataFilterExtensionUtil';
+import useGeoJsonFeatures from './geojson/useGeoJsonFeatures';
+import useTilesetFeatures from './tileset/useTilesetFeatures';
+import { getDataFilterExtensionProps } from './dataFilterExtensionUtils';
```

Comment:
Don't rename dataFilterExtensionUtil by dataFilterExtensionUtil`s`. It's break compatibility

Reply:
Updated.